### DISCUSSION TITLE
npm install fails

### DISCUSSION THREAD

**Discussion by yui1993** [tailwindlabs/tailwindcss#2163] [Help]
### Describe the problem:
NPM install fails whenever i run npm install tailwindcss i get these errors

73 silly fetchPackageMetaData error for resolve@^1.14.2 Unexpected end of JSON input while parsing near '...0","array.prototype.m'
74 timing stage:rollbackFailedOptional Completed in 3ms
75 timing stage:runTopLevelLifecycles Completed in 6280ms
76 verbose stack SyntaxError: Unexpected end of JSON input while parsing near '...0","array.prototype.m'
76 verbose stack     at JSON.parse (<anonymous>)
76 verbose stack     at parseJson (C:\Program Files\nodejs\node_modules\npm\node_modules\json-parse-better-errors\index.js:7:17)
76 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-fetch-npm\src\body.js:96:50
76 verbose stack     at processTicksAndRejections (internal/process/task_queues.js:97:5)
77 verbose cwd C:\Users\michael\Documents\tailwind-chat
78 verbose Windows_NT 10.0.19041
79 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "tailwindcss"
80 verbose node v12.18.3
81 verbose npm  v6.14.6
82 error Unexpected end of JSON input while parsing near '...0","array.prototype.m'
83 verbose exit [ 1, true ]

**Comment by gsimone** [comment]
try these steps:

1. npm cache clean --force
2. delete node_modules and package-lock.json
3. npm install
